RSA_test.c

```diff
@@ -118,14 +161,50 @@ void measure_performance() {
     average2 = total / NUM_TRIALS;
 
     result = average1 - average2;
-    if ( result < 0 )
-        result = 0;
 
     printf("\tAverage Ticks: %.2f cycles\n\n", average2);
-    printf("Results:\n");
+
+    printf("Constant Input Results:\n");
     // printf("\tClocks per second = %ld\n", CLOCKS_PER_SEC);
     printf("\tNumber of trials  = %d\n", NUM_TRIALS);
     printf("\tImproved by       = %.0f cycles", result);
     result = ( result / average1) * 100;
     printf(", [ %.1f%% ]\n\n", result);
+
+
+    printf("Unoptimized Implementation on File input:\n");
+
+    start = clock();
+    for ( int i=0; i < NUM_TRIALS; i++ ) {
+        initial_file_encrypt();
+        initial_file_decrypt();
+    }
+    end = clock();
+    total = end - start;
+    average1 = total / NUM_TRIALS;
+
+    printf("\tAverage Ticks: %.2f cycles\n\n", average1);
+
+    printf("Optimized Implementation on File input:\n");
+
+    start = clock();
+    for ( int i=0; i < NUM_TRIALS; i++ ) {
```

Comment:
I think I have to change the number of iterations here because this took a long time with file encryption/decryption